recipes/satsuma/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "satsuma2" %}
+{% set version = "20161123" %}
+{% set sha256 = "78e08ce7f72e839ab9bfbe3371cf08c93b6a1f208bcf9b0df9794202c269ba76" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: https://github.com/bioinfologics/satsuma2/archive/da694aeecf352e344b790bea4a7aaa529f5b69e6.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - gcc_linux-64  # [linux64]
```

Comment:
The normal `gcc` package is not working?

Reply:
gcc is too old.